exercise_08/formatStringByWordsLength.js

```diff
@@ -0,0 +1,35 @@
+import getTotalWordsFromString from "../exercise_02/getTotalWordsFromString.js";
+
+/**
+ * Write JS function "formatStringByWordsLength" to check number of words contained in given string
+    - if total number of words is greater then 5, then return uppercase string
+    - otherwise, return lower case string
+ * input: string (sentence) 
+ * output: string
+ */
+
+/**
+ * CASE #1
+ * input: "This is a long, long, long sentence."
+ * output: "THIS IS A LONG, LONG, LONG SENTENCE."
+ */
+
+/**
+ * CASE #2 * input: "This is short one."
+ * output: "this is short one."
+ */
+
+/**
+ * check number of words contained in given string
+    - if total number of words is greater then 5, then return uppercase string
+    - otherwise, return lower case string
+ *  @param {string} sentence
+ *  @returns string
+ */
+
+const formatStringByWordsLength = function (sentence) {
+  const numberWords = getTotalWordsFromString(sentence);
+  return numberWords > 5 ? sentence.toUpperCase() : sentence.toLowerCase();
```

Comment:
We defined a method to check wether a value is greater or not than a threshold.

FIX: use abstractions, not implementations.